games/g.yaml

```diff
@@ -62,7 +62,7 @@
 - name: Geometry Wars Remake
   lang: Java
   license:
-  - GPL
+  - Artistic
```

Comment:
I'm not sure about this license; this repo is in google code so it's not easy to check, but is there an actual license file? I wonder why it would be listed as "Artistic License/GPL" on the project page. It could be some default value, like the first option in a dropdown.

Reply:
Yeah, the Google Code default is Artistic/GPL, but no GPL version specification on the site. I think the Artistic license for the assetts and GPL for the code...

Reply:
If the license files cannot be found, this should be As-is

Reply:
Changed the Artistic licenses to As-is and modified the 3 games too, which was unable to determine the version.